### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by chengzeyi**

### CODE REVIEW

File: src/sfast/triton/torch_ops.py [hunk 1]
```diff
@@ -119,8 +119,10 @@ def forward(ctx, input, num_groups, weight=None, bias=None, eps=1e-05):
                     weight = weight.contiguous()
                 if bias is not None:
                     bias = bias.contiguous()
+                N, C = input.shape[:2]
+                HxW = input.numel() // (N * C)
                 output, mean, rstd = aten.native_group_norm(
-                    input, num_groups, weight, bias, eps)
+                    input, weight, bias, N, C, HxW, num_groups, eps)
```
Comment (jon-chuang) [inline src/sfast/triton/torch_ops.py:125]: Hmm I'm getting an error as though this change was not made. i.e. num_groups is still passed at pos 2.

Comment (jon-chuang) [reply src/sfast/triton/torch_ops.py:125]: I guess this didn't make it into release

Comment (chengzeyi) [reply src/sfast/triton/torch_ops.py:125]: Yes, not released yet.😂 Building a dynamo is not easy.

Comment (jon-chuang) [reply src/sfast/triton/torch_ops.py:125]: Yep, just checked that only the nightly release has this fix.

**Pull request merged**